src/main/scala/no/ndla/articleapi/model/domain/Article.scala

```diff
@@ -16,8 +16,8 @@ import org.json4s.FieldSerializer._
 import org.json4s.native.Serialization._
 import scalikejdbc._
 
-case class Article(id: Option[Long],
-                   revision: Option[Int],
+case class Article(_id: Option[Long],
+                   _revision: Option[Int],
```

Comment:
Ja, jeg synes jo ikke dette blir så pent. Jeg hadde sett for meg mer "standard" innkapsling og konstruktør-tankegang, med to konstruktører én med `id`og `revision` (som da ikke er `Option`) og én uten. Og gettere som ikke har `Option`. Men det er kanskje ikke mulig med en case class?

Hvis vi nå må la koden forholde seg til variable med `_` foran blir jo det litt pest og kolera, og jeg heller mot å være enig med @kstigen at dette ikke ble noe bedre.

Men jeg må jo si at hvis dette ikke er mulig i scala har jeg funnet første case (pun intended :) ) hvor java er bedre enn scala.